packages/components/pages/f-account-info/test/visual/f-account-info.visual.desktop.spec.js

```diff
@@ -30,74 +34,57 @@ describe('f-account-info - Desktop Visual Tests', () => {
         accountInfo.clickOutOfInputField();
 
         // Assert
-        browser.percyScreenshot(`f-account-info - ${field} Input Error Message`, PLATFORM);
+        browser.percyScreenshot(`f-account-info - ${field} Input Error Message`, device);
     });
 
     it('should display the illegal first name error message immediately on click', () => {
         // Arrange
-        const customerInput = {
-            firstName: {
-                input: '123'
-            }
-        };
+        const input = '123';
 
         // Act
         accountInfo.clearBlurField('firstName');
-        accountInfo.populateAccountForm('firstName', customerInput);
+        accountInfo.setFieldValue('firstName', input);
         accountInfo.clickOutOfInputField();
 
         // Assert
-        browser.percyScreenshot('f-account-info - illegal first name error message', PLATFORM);
+        browser.percyScreenshot('f-account-info - illegal first name error message', device);
     });
 
     it('should display the illegal last name error message immediately on click', () => {
         // Arrange
-        const customerInput = {
-            lastName: {
-                input: '123'
-            }
-        };
+        const input = '123';
```

Comment:
Probably worth making this a global var like the other spec for consistency and rename to `illegalInput`